client/src/components/SitterProfile/sendRequestCard/SendRequestCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { Typography, Box, Container, Paper, InputAdornment, TextField, Button } from '@material-ui/core';
+import useStyles from './useStyles';
+import Rating from '@material-ui/lab/Rating';
+import DateRangeIcon from '@material-ui/icons/DateRange';
+import { useSitters } from '../../../context/useSitterContext';
+import DatePicker from 'react-multi-date-picker';
+import TimePicker from 'react-multi-date-picker/plugins/time_picker';
+import { useRef } from 'react';
+
+export default function SendRequestCard(): JSX.Element {
+  const classes = useStyles();
+  const { selectedSitter } = useSitters();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const startDatePickerRef = useRef<any | null>();
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const endDatePickerRef = useRef<any | null>();
+  const fields = ['start', 'end'];
+  return (
+    <Box display="flex">
+      <Container className={classes.cardMain} component={Paper}>
+        <Box>
+          <Box display="flex" justifyContent="center" marginTop={2}>
+            <Typography className={classes.name}>{`$${selectedSitter?.hourlyRate}/hr`}</Typography>
+          </Box>
+          <Box display="flex" marginTop={1} marginBottom={2} justifyContent="center">
+            <Rating name="read-only" value={4} readOnly />
+          </Box>
+          <Box>
+            {fields.map((field) => {
```

Comment:
Good thinking! Please make sure each item has a unique key, using indexes as keys here is okay as we have a static list :)

Reply:
I ended up changing this to implementation to use props.